contrib/utilities/indent.py

```diff
@@ -0,0 +1,260 @@
+
```

Comment:
please copy deal.II copyright header (last time I added `.py` I copied the header verbatim and added comment character in editor).